bigtable/src/test/java/com/m/examples/bigtable/InstanceAdminExampleTest.java

```diff
@@ -51,9 +50,16 @@
   private String instanceId;
   private InstanceAdminExample instanceAdmin;
 
+  private static String requireEnv(String varName) {
+    assertNotNull(
+        System.getenv(varName),
+        "System property '%s' is required to perform these tests.".format(varName));
+    return System.getenv(varName);
+  }
+
   @BeforeClass
   public static void beforeClass() throws IOException {
-    projectId = System.getProperty(PROJECT_PROPERTY_NAME);
+    projectId = requireEnv("GOOGLE_CLOUD_PROJECT");
     if (projectId == null) {
```

Comment:
`requireEnv` should throw an error if `projectID` is null, so you can remove this check